modules/kn-trigger-filtering.adoc

```diff
@@ -1,9 +1,5 @@
-// Module included in the following assemblies:
-//
-// * /serverless/knative_eventing/serverless-using-brokers.adoc
-
 [id="kn-trigger-filtering_{context}"]
-= Filtering events using triggers
+= Filtering events using triggers with the Knative CLI
```

Comment:
Should you s/with/using to match the previous modules?

Reply:
Updated this heading to avoid having "using" twice